test/Mechanical/rotational.jl

```diff
@@ -8,215 +8,219 @@ using OrdinaryDiffEq: ReturnCode.Success
 # using Plots
 
 @testset "two inertias" begin
-    @named fixed = Fixed()
-    @named inertia1 = Inertia(J = 2) # this one is fixed
-    @named spring = Spring(c = 1e4)
-    @named damper = Damper(d = 10)
-    @named inertia2 = Inertia(J = 2, phi = pi / 2)
-
-    connections = [connect(fixed.flange, inertia1.flange_b)
-                   connect(inertia1.flange_b, spring.flange_a, damper.flange_a)
-                   connect(spring.flange_b, damper.flange_b, inertia2.flange_a)]
+    @mtkmodel TwoInertia begin
+        @components begin
+            fixed = Fixed()
+            inertia1 = Inertia(J = 2) # this one is fixed
+            spring = Spring(c = 1e4)
+            damper = Damper(d = 10)
+            inertia2 = Inertia(J = 2, phi = pi / 2)
+        end
+        @equations begin
+            connect(fixed.flange, inertia1.flange_b)
+            connect(inertia1.flange_b, spring.flange_a, damper.flange_a)
+            connect(spring.flange_b, damper.flange_b, inertia2.flange_a)
+        end
+    end
 
-    @named model = ODESystem(connections, t,
-        systems = [fixed, inertia1, inertia2, spring, damper])
-    sys = structural_simplify(model)
+    @mtkbuild sys = TwoInertia()
 
-    prob = ODEProblem(sys, Pair[], (0, 10.0))
+    prob = ODEProblem(sys, [D(D(sys.inertia2.phi)) => 0], (0, 10.0))
```

Comment:
@YingboMa this and in following ODE and DAE problems, now we have to pass an initial value to `sys₊inertia2₊phi_tt`. However, while adding this lib this was discussed and after some changes in tearing (in MTK), this was no longer required.
Why is it required again?